### ISSUE TITLE
util.inherits is not a function

### ISSUE THREAD

**Issue by ralyodio** [auth0/node-jsonwebtoken#825]
Getting this error in sveltekit:

```
500

util.inherits is not a function

node_modules/jws/lib/data-stream.js@http://localhost:3000/node_modules/.vite/deps/jsonwebtoken.js?v=0b51d446:77:10
__require2@http://localhost:3000/node_modules/.vite/deps/chunk-EVPD4D7C.js?v=0b51d446:47:50
node_modules/jws/lib/sign-stream.js@http://localhost:3000/node_modules/.vite/deps/jsonwebtoken.js?v=0b51d446:535:22
__require2@http://localhost:3000/node_modules/.vite/deps/chunk-EVPD4D7C.js?v=0b51d446:47:50
node_modules/jws/index.js@http://localhost:3000/node_modules/.vite/deps/jsonwebtoken.js?v=0b51d446:718:22
__require2@http://localhost:3000/node_modules/.vite/deps/chunk-EVPD4D7C.js?v=0b51d446:47:50
node_modules/jsonwebtoken/decode.js@http://localhost:3000/node_modules/.vite/deps/jsonwebtoken.js?v=0b51d446:753:15
__require2@http://localhost:3000/node_modules/.vite/deps/chunk-EVPD4D7C.js?v=0b51d446:47:50
node_modules/jsonwebtoken/index.js@http://localhost:3000/node_modules/.vite/deps/jsonwebtoken.js?v=0b51d446:2845:15
__require2@http://localhost:3000/node_modules/.vite/deps/chunk-EVPD4D7C.js?v=0b51d446:47:50
@http://localhost:3000/node_modules/.vite/deps/jsonwebtoken.js?v=0b51d446:2858:28

```

**Comment by townsean** [comment]
I ran into this issue I got a fix for it. I had to config Vite to add a polyfill for the util module.

From a comment on a [different GitHub issue](https://github.com/vitejs/vite/issues/6101#issuecomment-992814321), I learned builtin node modules aren't polyfilled by default with Vite so you'll need to [manually add them](https://www.thecodingcouple.com/uncaught-typeerror-util-inherits-is-not-a-function/).  This [Medium articl](https://medium.com/@ftaioli/using-node-js-builtin-modules-with-vite-6194737c2cd2)e was useful in figuring out how to do that.

**Comment by ralyodio** [comment]
yeah i tried that. it fixed that issue but not Buffer. It works in static build but not dev now. Dev says buffer is imported twice. I gave up and did a signing server on the backend.

**Comment by BasilaryGroup** [comment]
There is a very simple work around, use https://github.com/panva/jose

It look less than 2 minutes to install and modify my code.

**Comment by cryptominnow** [comment]
> townsean

[vite-plugin-node-polyfills](https://github.com/davidmyersdev/vite-plugin-node-polyfills#vite-plugin-node-polyfills) appears to be a better modern solution for vite users

**Comment by cryptominnow** [comment]
> I ran into this issue I got a fix for it. I had to config Vite to add a polyfill for the util module.
> 
> From a comment on a [different GitHub issue](https://github.com/vitejs/vite/issues/6101#issuecomment-992814321), I learned builtin node modules aren't polyfilled by default with Vite so you'll need to [manually add them](https://www.thecodingcouple.com/uncaught-typeerror-util-inherits-is-not-a-function/). This [Medium articl](https://medium.com/@ftaioli/using-node-js-builtin-modules-with-vite-6194737c2cd2)e was useful in figuring out how to do that.

[vite-plugin-node-polyfills](https://github.com/davidmyersdev/vite-plugin-node-polyfills#vite-plugin-node-polyfills) appears to be a better modern solution for vite users

**Comment by Alan-Graton** [comment]
> I ran into this issue I got a fix for it. I had to config Vite to add a polyfill for the util module.
> 
> From a comment on a [different GitHub issue](https://github.com/vitejs/vite/issues/6101#issuecomment-992814321), I learned builtin node modules aren't polyfilled by default with Vite so you'll need to [manually add them](https://www.thecodingcouple.com/uncaught-typeerror-util-inherits-is-not-a-function/). This [Medium articl](https://medium.com/@ftaioli/using-node-js-builtin-modules-with-vite-6194737c2cd2)e was useful in figuring out how to do that.

You saved me dude, thank you very much ^-^
